content/blog/index.njk

```diff
@@ -6,7 +6,7 @@ action:
   text: Browse by tag »
   url: /tags/
 pagination:
-  data: collections._post
+  data: collections.posts
```

Comment:
@mirisuzanne Because we use a custom sort (by `end` date), this needs to be sorted before pagination -- without this, the TimeDesigner case study was paginated with 2017 posts, but displayed as a 2022 post.